pages/newsletter/index.js

```diff
@@ -13,7 +13,7 @@ export default function NewsletterIndexPage() {
       </StickyNavbar>
       <div className="py-12 mt-12">
         <Container wide>
-          <NewsletterSubscribe formName="form 1" />
+          <NewsletterSubscribe />
```

Comment:
@fmvilas I am slightly concerned about why we are making the newsletter a separate route on the website. We aren't using this route anywhere in the website, not in the navbar or footer as well. We can simply have `NewsletterSubscribe` as a component and can be used in various places as needed. So, can we remove this file?

Reply:
@akshatnema this is meant to be referenced externally (emails, slides, etc.). It's a landing page.

Reply:
@akshatnema it was there before, it is for tweets and other stuff that @fmvilas mentioned. So you do not send people to asyncapi.com and make them look for newsletter subscription component, but just send them to https://asyncapi.com/newsletter